tests/protractor/e2e/registration-by-sms.js

```diff
@@ -0,0 +1,279 @@
+const utils = require('../utils');
+
+describe('registration transition', () => {
+
+  'use strict';
+
+  const PHONE = '+64271234567';
+  const FORM_NAME = 'Registration';
+
+  // contacts
+  const BOB_PLACE = {
+    _id: 'bob-contact',
+    reported_date: 1,
+    type: 'clinic',
+    name: 'Bob Place'
+  };
+  const CAROL = {
+    _id: 'carol-contact',
+    reported_date: 1,
+    type: 'person',
+    phone: PHONE,
+    name: 'Carol Carolina',
+    parent: BOB_PLACE,
+    patient_id: '05946',
+    sex: 'f',
+    date_of_birth: 1462333250374
+  };
+
+  const DOCS = [ BOB_PLACE, CAROL ];
+  const CONFIG = {
+    transitions: {
+      accept_patient_reports: {
+        load: './transitions/accept_patient_reports.js'
```

Comment:
Do we actually need this load stuff? I thought these days you just make the object attached to the property truthy and Sentinel did the rest (because they are all whitelisted, so you can't load arbitrary transitions anyway)